src/app/routes/getInitialData/index.js

```diff
@@ -1,27 +1,40 @@
 import 'isomorphic-fetch';
 
+const upstreamStatusCodesToPropagate = [200, 404, 502];
```

Comment:
```suggestion
const upstreamStatusCodesToPropagate = [200, 404];
```
Keeps the added benefit of the `Unexpected upstream response` console warning

Reply:
Good shout. Thanks!